packaging/rpm/microshift.spec

```diff
@@ -184,14 +184,17 @@ install -d -m755 %{buildroot}/%{_unitdir}
 install -p -m644 packaging/systemd/microshift.service %{buildroot}%{_unitdir}/microshift.service
 
 install -d -m755 %{buildroot}/%{_sysconfdir}/microshift
+install -d -m755 %{buildroot}/%{_sysconfdir}/microshift/spec
 install -d -m755 %{buildroot}/%{_sysconfdir}/microshift/manifests
 install -d -m755 %{buildroot}/%{_sysconfdir}/microshift/manifests.d
+install -p -m644 cmd/generate-config/config/config-openapi-spec.json %{buildroot}%{_sysconfdir}/microshift/spec/config-openapi-spec.json
 install -p -m644 packaging/microshift/config.yaml %{buildroot}%{_sysconfdir}/microshift/config.yaml.default
 install -p -m644 packaging/microshift/lvmd.yaml %{buildroot}%{_sysconfdir}/microshift/lvmd.yaml.default
 install -p -m644 packaging/microshift/ovn.yaml %{buildroot}%{_sysconfdir}/microshift/ovn.yaml.default
 
 # /usr/lib/microshift manifest directories for other packages to add to
 install -d -m755 %{buildroot}/%{_prefix}/lib/microshift
+install -d -m755 %{buildroot}/%{_prefix}/lib/microshift/spec
```

Comment:
I think this type of read-only files are better suited with in the `%{buildroot}%{_datadir}/microshift` directory.

Reply:
We already have release*.json and greenboot functions there.

Reply:
Sure thing Giga, updated. Quick clarification, did you still want to keep the `spec` folder?